dynamic-plugins.default.yaml

```
@@ -712,3 +712,15 @@ plugins:
                   props:
                     width: 1500
                     height: 800
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment
+    disabled: true
+    pluginConfig:
+      dynamicPlugins:
+        frontend:
+          janus-idp.backstage-plugin-analytics-provider-segment:
+            apiFactories:
+              - importName: SegmentAnalyticsApi
+      app:
+        analytics:
+          segment:
+            writeKey: ${SEGMENT_WRITE_KEY}
```

shall we add other options as well?

```suggestion
            writeKey: ${SEGMENT_WRITE_KEY}
            maskIP: ${SEGMENT_MASK_IP} # prevents IP addresses from being sent if true
            testMode: ${SEGMENT_TEST_MODE} # prevents data from being sent if true
```